src/components/i18n-helpers.tsx

```diff
@@ -30,3 +32,27 @@ export const useLocaleFormat = (dateFormat) => {
     }
   };
 };
+
+export const useTranslationParagraphs = (
+  ns?: string,
+  options?: UseTranslationOptions,
+) => {
+  const { t, ...useTranslationResponse } = useTranslation(ns, options);
+
+  const tWithParagraphs = (key) => {
+    const texts = t(key).split(/\n\s*\n/);
+
+    return texts.map((text) => {
+      const textWithBreaks = text
+        .split(/\n/)
+        .map((textContent) => [
+          textContent,
+          <br key={textContent.substring(0, 7)} />,
+        ])
+        .flat();
+      return <p key={text.substring(0, 7)}>{textWithBreaks}</p>;
```

Comment:
You calculate the same id for two different elements. The purpose of the key is to recognize the elements after a re-rendering but with two ids this will not happen. Maybe a counter is again easier?